yt_dlp/extractor/bilibili.py

```diff
@@ -80,42 +79,20 @@ class BiliBiliIE(InfoExtractor):
         },
         'skip': 'Geo-restricted to China',
     }, {
-        # Title with double quotes
         'url': 'http://www.bilibili.com/video/av8903802/',
         'info_dict': {
-            'id': '8903802',
-            'title': '阿滴英文｜英文歌分享#6 "Closer',
+            'id': '8903802_part1',
+            'ext': 'mp4',
+            'title': '161020 Closer  ',
+            'upload_date': '20170301',
             'description': '滴妹今天唱Closer給你聽! 有史以来，被推最多次也是最久的歌曲，其实歌词跟我原本想像差蛮多的，不过还是好听！ 微博@阿滴英文',
+            'timestamp': 1488382634,
+            'uploader_id': '65880958',
+            'uploader': '阿滴英文',
+        },
+        'params': {
+            'skip_download': True,
         },
-        'playlist': [{
-            'info_dict': {
-                'id': '8903802_part1',
-                'ext': 'flv',
-                'title': '阿滴英文｜英文歌分享#6 "Closer',
-                'description': 'md5:3b1b9e25b78da4ef87e9b548b88ee76a',
-                'uploader': '阿滴英文',
-                'uploader_id': '65880958',
-                'timestamp': 1488382634,
-                'upload_date': '20170301',
-            },
-            'params': {
-                'skip_download': True,
-            },
-        }, {
-            'info_dict': {
-                'id': '8903802_part2',
-                'ext': 'flv',
-                'title': '阿滴英文｜英文歌分享#6 "Closer',
-                'description': 'md5:3b1b9e25b78da4ef87e9b548b88ee76a',
-                'uploader': '阿滴英文',
-                'uploader_id': '65880958',
-                'timestamp': 1488382634,
-                'upload_date': '20170301',
-            },
-            'params': {
-                'skip_download': True,
-            },
-        }]
```

Comment:
there is no longer any anthology in the tests?

Reply:
No, there is no anthology.

Reply:
So the `_part1` suffix and the anthology code can be removed?

Reply:
I don't relay know, I don't use the service or understand Chinese, but from what I have been able to tell it seems like they no longer exist I would leave anthology code because I'm not 100% sure.

Reply:
Oh, ok. You can ignore this then. I was under the impression you are familiar with the site